sensor/EventSensor.js

```diff
@@ -0,0 +1,106 @@
+/*    Copyright 2018 Firewalla LLC
```

Comment:
need to update copyrights for every file: `Copyright 2020 Firewalla INC.`

Reply:
updated